### DISCUSSION TITLE
Acoular Workshop 2024

### DISCUSSION THREAD

**Discussion by adku1173** [acoular/acoular#224] [Announcements]
We are pleased to announce the **3rd workshop on Acoular** :tada: . 

The workshop, which will be held **June 12-13, 2024, at the TU Berlin Department of Engineering Acoustics**, is **free of charge** and suitable for both first-time and experienced Acoular users. 

The two-day event explores frequency and time domain methods, guiding participants from fundamental processing chains to hands-on implementation of custom beamforming algorithms. Beyond basic usage, this workshop offers a unique opportunity for actual array experiments with Acoular. More **details on the workshop program can be found** [here](https://www.tu.berlin/en/event-details/events/event/018c1020-419d-70b6-abcf-2220015bd233)

To secure your spot at the workshop **register here**: [Acoular Workshop Registration](https://events.tu-berlin.de/de/events/018c1020-419d-70b6-abcf-2220015bd233/apply)

**Comment by esarradj** [comment]
For attendees: https://tubcloud.tu-berlin.de/s/igAeLdNo2WSRWEp
